x-pack/plugins/security_solution/server/usage/detections/rules/get_metrics.mocks.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SearchResponse } from '@elastic/elasticsearch/lib/api/types';
+
+export const getMockRuleAlertsResponse = (docCount: number): SearchResponse<unknown, unknown> => ({
```

Comment:
You will know this better than me, but is there not a shared rule client for the security solution so everyone / everything interfaces with the data in the same way? Same question for mocks?

Reply:
In most cases, yeah, we start collapsing types where we can. Reducing from duplicate types is tricky. I changed most of the others. I can see if I just missed this one or not.

Depending on where you're getting the type from (REST API, Elastic SO, Elastic data set, etc...) They contain in some cases different shapes such as snake_case vs. camelcase.

I will check this one though and see I can give it a stronger type for you.